docs/document-level-security/NETWORK_DRIVE.md

```diff
@@ -0,0 +1,27 @@
+### Setting up the Network Drive connector
+
+See the [Developer guide](../../docs/DEVELOPING.md) for setting up connectors.
+
+## Document level security
+
+Document Level Security (DLS) enables you to restrict access to documents based on a user's permissions.
+This feature is available by default for the Network Drive connector.
+DLS in the Network drive connector facilitates the syncing of folder & file permissions, including both user and group level permissions.
+
+The native Network Drive connector will offer DLS support for Windows Network Drive only. DLS support for Linux network drive will be available via self-managed connector client.
```

Comment:
It's still not explained how I would do this, or what would need to be set up.

Reply:
I completely agree with your point. However, in the current stage of our discussion, we do not yet have the code for Linux network drive support. This topic is still under discussion, and the implementation code has not yet undergone review.

As a resolution, we have two options:

1. Put this PR on hold until the implementation part of Linux Network Drive is reviewed, and we can revisit this PR at a later time.

2. Remove the mention of Linux Network Drive support from this Documentation PR or keep it open-ended. Further, raise a PR to add the code for Linux Network Drive support and changes in the Documentation with detailed guidelines on how to make it work.

Please let me know which of these options you think is the best way forward, or if you have any other suggestions."

Reply:
> Remove the mention of Linux Network Drive support from this Documentation PR or keep it open-ended. Further, raise a PR to add the code for Linux Network Drive support and changes in the Documentation with detailed guidelines on how to make it work.

This would be my vote. That way, if we don't come up with a solution before release, we don't have to remember to come back and edit docs to take out the reference.